internal/logger/logger.go

```diff
@@ -93,6 +93,8 @@ type FunLogger struct {
 	Fail chan struct{}
 	// Wg is a WaitGroup that can be used to wait for the loading animation to finish.
 	Wg *sync.WaitGroup
+	// CI is a boolean that is set to true if the logger is running in a CI environment.
+	CI bool
```

Comment:
Should this be `IsCI`? or something "boolean"?

Reply:
I am using `CI` because is a default ENVVAR set by GitHub actions - https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables

Reply:
Yes, but this is not an envvar. This is a go member.